stdlib/3.4/asyncio/queues.pyi

```diff
@@ -32,15 +33,19 @@ class Queue(Generic[T]):
     @coroutine
     def get(self) -> Future[T]: ...
     def get_nowait(self) -> T: ...
+    if sys.version_info >= (3, 4, 4):
```

Comment:
My preference is to consider `join()` and `task_done()` valid for any version >= 3.4. A user on 3.4.4 should not be penalized for using 3.4.4 properly. A user on 3.4.0--3.4.3 should not be using the `join()` and `task_done()` methods anyway.

We'll have the same issue with `JoinableQueue` which was deprecated in 3.4.4. My preference in that case is to consider it valid for any version < 3.5. A user on 3.4.0--3.4.3 should not be penalized for type checking a `JoinableQueue` whereas a user on 3.4.4 should know not to use the class.

I'll update this PR accordingly.